pkg/tools/dnsutils/dnsconfigs/tcp_dns_server_test.go

```diff
@@ -35,11 +35,17 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/tools/dnsutils"
 	"github.com/networkservicemesh/sdk/pkg/tools/dnsutils/dnsconfigs"
 	"github.com/networkservicemesh/sdk/pkg/tools/dnsutils/fanout"
+	"github.com/networkservicemesh/sdk/pkg/tools/dnsutils/memory"
 	"github.com/networkservicemesh/sdk/pkg/tools/dnsutils/next"
 	"github.com/networkservicemesh/sdk/pkg/tools/dnsutils/noloop"
 	"github.com/networkservicemesh/sdk/pkg/tools/dnsutils/searches"
 )
 
+const (
+	resolvedIP      = "1.1.1.1"
```

Comment:
I'd prefer to use different inline ips for the test check and the health check to reduce complexity.

Reply:
Fixed